README.md

```diff
@@ -8,6 +8,46 @@
+* [Compute Canada: Graham](https://docs.alliancecan.ca/wiki/Graham) - An introduction of Graham, a heterogeneous cluster located at the University of Waterloo.
+* [Compute Canada: Narval](https://docs.alliancecan.ca/wiki/Narval/en) - An introduction of Narval, a general purpose cluster located at the École de technologie supérieure in Montreal.
+* [Compute Canada: Béluga](https://docs.alliancecan.ca/wiki/B%C3%A9luga/en) - An introduction of Béluga, a general purpose cluster situated at the École de technologie supérieure in Montreal.
+* [Compute Canada: Niagara](https://docs.alliancecan.ca/wiki/Niagara) - An introduction of Niagara, a homogeneous cluster owned by the University of Toronto and operated by SciNet.
+* [Compute Canada: Lustre](https://docs.alliancecan.ca/wiki/Tuning_Lustre) - An introduction and tutorial of Lustre Filesystem.
+* [Compute Canada: Storage and file management](https://docs.alliancecan.ca/wiki/Storage_and_file_management) - An introduction of a wide range of storage options to cover the needs of very diverse users of Compute Canada.
+* [Ubuntu HPC](https://www.youtube.com/watch?v=tGIobcyKViI&t=2s) - A Youtube video introducting cluster components.
+* [Top 500 list](https://www.top500.org/) - The data of top 500 listed supercomputers, which gives a scope of how fast and how much compute nodes are, and also introduction and comparison among these computers.
+* [Iowa State University HPC Guides](https://www.hpc.iastate.edu/guides/introduction-to-hpc-clusters) - A collection of introductory guides for HPC including SLURM, globus, unix, python, julia, and much more.
+* [An HPC User Guide](https://servicedesk.aub.edu.lb/TDClient/1398/Portal/KB/ArticleDet?ID=66391) - A collection of introductory guides for HPC including connection nodes, SLURM, job examples and applications.
+
+## When should I use a cluster?
+
+* [Compute Canada: Systems available](https://docs.alliancecan.ca/wiki/Getting_started#What_systems_are_available?) - A brief introduction for the five systems of Compute Canada and the clusters' types.
+* [Compute Canada: Allocations and compute scheduling](https://docs.alliancecan.ca/wiki/Allocations_and_compute_scheduling#Choosing_GPU_models_for_your_project) - A brief tutorial of choosing GPU models for your project.
+* [Compute Canada: Checkpoints](https://docs.alliancecan.ca/wiki/Points_de_contr%C3%B4le/en) - A tutorial of creating and loading a checkpoint.
+* [Compute Canada: Machine Learning tutorial](https://docs.alliancecan.ca/wiki/Tutoriel_Apprentissage_machine/en#Checkpointing_a_long-running_job) - A toturial of checkpointing a long-running job.
+* [Tensorflow: Checkpoint guide](https://www.tensorflow.org/guide/checkpoint) - Guidance for training checkpoints while using Tensorflow.
+* [Enabling Coordinated Checkpointing for Distributed HPC Application](https://www.youtube.com/watch?v=lswkWQU4_W0) - A Youtube video presenting a new coordination mechanism for Checkpoint/Restore in User-Space.
```

Comment:
Looks like this is a kubernetes resource, and we won't be using kube for any of the clusters. We can remove this resource, I think